src/Nest/XPack/MachineLearning/MlInfo/ElasticClient-MlInfo.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+
+namespace Nest
+{
+	public partial interface IElasticClient
+	{
+		/// <summary>
+		/// Retrieves machine learning job results for one or more categories.
+		/// </summary>
+		IMlInfoResponse MlInfo(Func<MlInfoDescriptor, IMlInfoRequest> selector = null);
```

Comment:
My vote would be to use `MachineLearningInfo` here

Reply:
I had also thought the same thing, however all of the other ML APIs use the `ML` prefix, so we would be breaking consistency. My vote would be to keep as is, can we get a tie breaking vote from @mpdreamz?

Reply:
The ML prefix is only in the specs though, not in the client source?

Reply:
We do have vague plans to move all of the ml api's behind a `MachineLearning` property in 7.0 as well.

I'd prefer `MachineLearningInfo()` for 6.x too.

Reply:
Done, renamed as voted! Thanks!